### DISCUSSION TITLE
Webhook log/transmission data

### DISCUSSION THREAD

**Discussion by k9protection** [capcom6/android-sms-gateway#92] [Ideas]
Maybe I'm being silly, but I can't see this at the moment.

Is there a way to see when a webhook attempts to transmit data to our server from within the android application?

I can look at the server logs on the firewall, but it doesn't give us any meaningful information while we're debugging.

**Comment by capcom6** [comment]
Unfortunately, there are no logs for webhook transmissions available at the moment. The only indication of a webhook attempt to send is a notification icon in the status bar of the app. However, the data transmission to the server can happen so quickly that the icon may only flash briefly.

Webhooks are a relatively new feature, so I'm still collecting feedback on it and plan to make improvements in the coming versions. Adding some form of logging is on my backlog.
